projects/schematics/src/migrations/mechanism/update-ssr/update-ssr-files.ts

```diff
@@ -4,8 +4,42 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { chain, Rule } from '@angular-devkit/schematics';
+import {
+  chain,
+  Rule,
+  SchematicContext,
+  Tree,
+} from '@angular-devkit/schematics';
+import { Path } from '@angular-devkit/core';
+import {
+  SERVER_BAK_FILENAME,
+  SERVER_FILENAME,
+} from '../../../shared/constants';
 
 export function updateServerFiles(): Rule {
-  return chain([]);
+  return chain([removeServer]);
+}
+
+function removeServer(): Rule {
+  return (tree: Tree, _context: SchematicContext) => {
+    let serverPath;
+    let serverBakPath;
+
+    tree.visit((filePath: Path) => {
+      const fileName = filePath.replace(/^.*[\\/]/, '');
+      if (fileName === SERVER_FILENAME) {
+        serverPath = filePath;
+      }
+
+      if (fileName === SERVER_BAK_FILENAME) {
+        serverBakPath = filePath;
+      }
+    });
```

Comment:
`nitpick:` Would be great to break the loop as soon as we find both paths. If there is no easy way to do it, let's leave it as it is.

Reply:
I've already thought about it and couldn't find an easy solution.